vernac/vernacentries.ml

```diff
@@ -1882,7 +1884,7 @@ let vernac_show = let open Feedback in function
       Constrextern.with_implicits msg_notice (pr_nth_open_subgoal n)
   | ShowProof -> show_proof ()
   | ShowNode -> show_node ()
-  | ShowScript -> Stm.show_script ()
+  | ShowScript -> (* Stm.show_script () *) ()
   | ShowExistentials -> show_top_evars ()
```

Comment:
I would do something here (at worst a hook, even temporary) so that the commit is without loss of functionality.